core/src/main/java/org/polypheny/db/config/RuntimeConfig.java

```diff
@@ -62,9 +62,9 @@ public enum RuntimeConfig {
             ConfigType.BOOLEAN
     ), // Druid
 
-    RELATIONAL_CASE_SENSITIVE(
+    NAMESPACE_DEFAULT_CASE_SENSITIVE(
```

Comment:
Put relational in the name.

Reply:
Adjusted.

Reply:
Is used independent of data model type, separated into different RuntimeConfig and adjusted call and description accordingly.